documentation/src/main/asciidoc/topics/con_store_migrator.adoc

```diff
@@ -0,0 +1,14 @@
+[id='store_migrator']
+= Cache Store Migrator
+{brandname} provides the `StoreMigrator.java` utility that reads data from
+cache stores from previous versions and rewrites it for compatibility with the
+latest {brandname} version.
```

Comment:
"from a previous version's cache store and recreates it using the latest {brandname} implementation" ?